src/payment/router.ts

```diff
@@ -9,7 +9,9 @@ async function makePayment(req: RequestWithUser, res: Response){
   const { destination, amount } = req.body;
```

Comment:
```suggestion
  const { destination: string, amount: string } = req.body;
```

Reply:
This is not type declaration but variable alias, it should be `{ destination, amount } : {destination: string, amount: string}`